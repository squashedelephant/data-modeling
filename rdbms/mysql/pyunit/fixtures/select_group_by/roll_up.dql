SELECT year, division, SUM(sales) AS annual_sales FROM forecast GROUP BY year ASC, division ASC WITH ROLLUP;
