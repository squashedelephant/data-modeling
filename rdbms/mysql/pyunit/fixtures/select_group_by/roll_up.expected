2017L
US East
280000.00
2017L
US Midwest
180000.00
2017L
US West
395000.00
2017L
None
855000.00
None
None
855000.00
